### PR TITLE
CE-299: add alt-text descriptions to the Ideas page

### PR DESCRIPTION
**Pull request by ChrisC**
### Resolves:

Partially addresses #7178 

### Changes:

Adds image descriptions to non-decorative images throughout the `/ideas` page. Also mutes decorative icons in various buttons throughout the Ideas page.

### Test Coverage:

No added tests but verified that this does not break existing tests.

### CODE REVIEW

**Comment by ChrisC** [comment]
h/t to @alflennik for support on generating some of the more challenging alternative-text descriptions here!!

File: src/views/ideas/ttt.json [hunk 1]
```diff
@@ -66,7 +80,9 @@
         "title": "ideas.animateACharacterTitle",
         "description": "ideas.animateACharacterDescription",
         "thumbImage": "/images/ideas/activities/animate-a-character-thumb.jpg",
+        "thumbImageDescription": "ideas.animateACharacterDescription",
```
Comment (corihudson) [inline src/views/ideas/ttt.json:83]:
```suggestion
        "thumbImageDescription": "ideas.animateACharacterImageDescription",
```

File: src/views/ideas/ttt.json [hunk 2]
```diff
@@ -66,7 +80,9 @@
...
         "modalImage": "/images/ideas/activities/animate-a-character-modal.jpg",
+        "modalImageDescription": "ideas.animateACharacterDescription",
```
Comment (corihudson) [inline src/views/ideas/ttt.json:85]:
```suggestion
        "modalImageDescription": "ideas.animateACharacterImageDescription",
```

**Review by corihudson — CHANGES_REQUESTED** [review]
Spotted one tile where the image alt text wasn't pointing to the right l10n string, but otherwise looks great, thank you!

**Comment by ChrisC** [comment]
@corihudson Thanks for spotting those translation key errors! Made the suggested changes 👍🏽

**Review by corihudson — APPROVED** [review]

**Pull request merged**